helm/hpcc/templates/dali.yaml

```diff
@@ -20,6 +20,7 @@ spec:
       labels:
         run: {{ $dali.name | quote }}
         app: dali
+        helmVersion: 8.2.0-closedown
```

Comment:
since this is auto replaced, and I assume we're not going to manually change this tag in each template each point/minor/major release, can we make these something generic like:
```
helmVersion: REPLACED_WHEN_PUBLISHED
```

?

Reply:
In theory we will automatically change this tag in each template file each release...

Reply:
ok yes I see, the change is checked back in.